code/modules/client/preference/link_processing.dm

```diff
@@ -29,7 +29,7 @@
 				var/datum/job/job = locate(href_list["job"])
 				if(job)
 					var/choices = list(job.title) + job.alt_titles
-					var/choice = input("Pick a title for [job.title].", "Character Generation", active_character.GetPlayerAltTitle(job)) as anything in choices | null
+					var/choice = tgui_input_list(user, "Pick a title for [job.title]", "Character Generation", choices)
```

Comment:
I'm unsure about all the stuff in this file, since its OOC preferences and I'm not sure if we want these TGUI'd yet. I'd say ask design team about these